src/main/java/org/opensearch/security/configuration/SecurityIndexSearcherWrapper.java

```diff
@@ -108,7 +116,8 @@ public final DirectoryReader apply(DirectoryReader reader) throws IOException {
             return new EmptyFilterLeafReader.EmptyDirectoryReader(reader);
         }
 
-        if (systemIndexEnabled && isBlockedSystemIndexRequest() && !isAdminDnOrPluginRequest()) {
+        if (systemIndexEnabled && isBlockedSystemIndexRequest() && !isAdminDnOrPluginRequest()
```

Comment:
Nit: What do you think about either combining isPermittedOnSystemIndex with isBlockedSystemIndexRequest or renaming the new check? From a code-reading perspective, I think it will be confusing to have conditions with such similar names.

Reply:
I'm combined isPermittedOnSystemIndex with isBlockedSystemIndexRequest.